docs/security/secure-data.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Securing your data
+description: How customer data is encrypted and how users can delete their personal data.
+---
+
+# Securing your data
+
+To ensure that you can securely manage your data when you use IBM Quantum&reg; Platform, it is important to know exactly what data is stored and encrypted and how you can delete any stored data.
+
+## Protecting your sensitive data
+
+The data that you store in IBM Cloud&reg; is encrypted at rest by using a randomly generated key.
+
+
+## Deleting your data
+
+Deleting a service instance removes all of the content associated with that instance, such as your jobs, results, parameters, and programs. To delete an instance, from the [Instances page,](https://quantum.cloud.ibm.com/instances) find the instance you want to remove, click its overflow menu, then click **Delete**. You will be asked to confirm the deletion.
+
```

Comment:
Is this the only way to delete data? I think you can remove it per-job too; right?  And what if you delete a user?  Are all of their instances (and related data) removed?